[ayucloud.services]
auth_path  user
auto_invite  F
buy  pass
email  hgsrxrbgr198@lyx13.xyz
last_checkin  2025-04-12 01:12:30.857548+08:00
name  AyucLouD-Services
node_n  99
sub_info  0B  0B  2025-04-13 01:12:30+08:00  (剩余 0B -1 day, 11:58:12.647227)
sub_url  https://network.ayucloud.nl/link/KCrCrK9fbi9PnI9K?clash=1
time  2025-04-12 01:12:31.869776+08:00
type  sspanel
保存base64/clash订阅失败  'NoneType' object is not iterable
尝试签到失败  HTTPSConnectionPool(host='ayucloud.services', port=443): Max retries exceeded with url: /user/login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fd2c8c61c60>, 'Connection to ayucloud.services timed out. (connect timeout=30)'))
更新订阅链接/续费续签失败  注册失败(avxnxcxn979@gmail.com): HTTPSConnectionPool(host='ayucloud.services', port=443): Max retries exceeded with url: /user/register (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fd2c8c616f0>, 'Connection to ayucloud.services timed out. (connect timeout=30)'))

[qlgq.top]

[cyanmori.com]

[vipxgyun.com]
